onnxruntime/test/onnx/onnxruntime_event.h

```diff
@@ -2,13 +2,13 @@
 // Licensed under the MIT License.
 
 #include <mutex>
-#include <pthread.h>
+#include <condition_variable>
 #include <core/common/common.h>
 
 struct OnnxRuntimeEvent {
  public:
-  pthread_mutex_t finish_event_mutex = PTHREAD_MUTEX_INITIALIZER;
-  pthread_cond_t finish_event_data = PTHREAD_COND_INITIALIZER;
+  std::mutex finish_event_mutex;
```

Comment:
why not nsync? i know it's test code, but for the sake of consistency.

Reply:
I haven't finished the pthread_cond_t part. Let's do it latter.

Reply:
sure, please mark a TODO. we should strive to be consistent in our code.